tests/functionaltests/FunctionalTests.cpp

```diff
@@ -643,3 +643,25 @@ class UIApplicationTests {
         UIApplicationTestsOpenURL();
     }
 }; /* class UIApplicationTests */
+
+
+//
+// UIApplicationTests
+//
+extern void NSURLStorageFileURL();
+
+class NSURLStorageFileTests {
+public:
+    BEGIN_TEST_CLASS(NSURLStorageFileTests)
+    TEST_CLASS_PROPERTY(L"RunAs", L"UAP")
+    TEST_CLASS_PROPERTY(L"UAP:Host", L"Xaml")
+    END_TEST_CLASS()
+
+    TEST_CLASS_SETUP(NSURLStorageFileTestsSetup) {
+        return SUCCEEDED(FrameworkHelper::RunOnUIThread(&UIApplicationDefaultInitialize));
+    }
+
```

Comment:
agreed; this requires cleanup.

---
In reply to: [84942793](https://github.com/Microsoft/WinObjC/pull/1215#discussion_r84942793) [](ancestors = 84942793)